sections/exchange/hooks/useExchange.tsx

```diff
@@ -141,6 +141,7 @@ const useExchange = ({
 	const [selectQuoteTokenModalOpen, setSelectQuoteTokenModalOpen] = useState<boolean>(false);
 	const [selectBaseTokenModalOpen, setSelectBaseTokenModalOpen] = useState<boolean>(false);
 	const [txApproveModalOpen, setTxApproveModalOpen] = useState<boolean>(false);
+	const [atomicExchangeSlippage] = useState<string>('0.01');
```

Comment:
Can this be a constant? Not sure if we need to make this state, since it's not changing at any point.

Reply:
Yes. This is a quick fix so we used the hardcoded value but we may change the slippage rate as a user-defined value later.
![image](https://user-images.githubusercontent.com/4819006/168894236-48bf9576-5b1e-4b00-b387-d7a2d1c5b2c5.png)

Reply:
Okay makes sense then.